perun-openapi/openapi.yml

```diff
@@ -8480,6 +8480,21 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/usersManager/getUserExtSourceByExtLoginAndExtSourceName:
+    get:
+      tags:
+        - UsersManager
+      operationId: getUserExtSourceByExtLoginAndExtSourceName
+      summary: Returns user's external source by the user's external login and external source name.
+      parameters:
+        - $ref: '#/components/parameters/extSourceName'
+        - $ref: '#/components/parameters/extLogin'
```

Comment:
The definition of extLogin in openApi uses name `extLogin`, but the method in RPC requires parameter called `extSourceLogin`, which would not match. I think you can write inline definition for the parameter instead of the used reference.